blog/articles/get-started-knative-eventing.md

```diff
@@ -0,0 +1,93 @@
+| Title  | LinkTitle | Date | Description | Type | Aliases
+| ------------- | ------------- | ------------- | ------------- | ------------- | ------------- | 
+| Getting started with Knative Eventing using Bitcoin transaction data  | Get Started with Knative Eventing  | 2020-05-01  | Guided walkthrough on using Knative Eventing  | Blog  | /blog/2010-05-01-getting-started-knative-evetning  |
+
+I’ve been exploring [Knative Eventing](https://knative.dev/docs/eventing/), a system that enables a cloud native eventing ecosystem to be easily deployed through the use of **event producers** and **event consumers.**
```

Comment:
Before we get into "I", we should have a [byline](https://www.lexico.com/en/definition/byline) and probably a bio.

Reply:
good point @evankanderson 
I actually opened [an issue](https://github.com/knative/docs/issues/2510) for this and forgot to tag you.